src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -222,6 +222,13 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
                 return e.message;
             }
         """.trimMargin())
+
+        // Let's wait till we are alone in the room
+        // (give time for the js Promise to finish before quiting selenium)
+        WebDriverWait(driver, 2).until {
```

Comment:
is there something specific driving this change?

Reply:
Yes, when using sipgw you see the participant and leave does not work. It is not noticeable with the recorder as the participant is not visible.